pages/home.html

```diff
@@ -84,7 +193,7 @@ <h5 class="text-center text-muted">Read the latest news about the Department of
         {%- endif -%}
     </div>
 
-
+    <!-- Events -->
```

Comment:
Can we use the same button like below in here? 

![image](https://github.com/user-attachments/assets/db545316-d5bd-4973-9c90-ccd2312701d7)